File: web/filing/models/income/house-property/house-property-loan-model.ts
export interface HousePropertyLoanModel {
    loanId?: number | null
    filingId?: number | null
    propertyId?: number | null
    vendorType?: string | null
    lenderName?: string | null
    loanAccountNumber?: string | null
    loanSanctionDate?: Date | null
    totalLoanAmount?: number | null
    loanOutstanding?: number | null
    interestPaid?: number | null
    principalRepaid?: number | null
  }

File: api/filing/models/house_property_loan_model.py
from datetime import date
from .api_base_model import ApiBaseModel


class HousePropertyLoanModel(ApiBaseModel):
    loan_id: int | None = None
    filing_id: int
    property_id: int | None = None
    vendor_type: str | None = None
    lender_name: str | None = None
    loan_account_number: str | None = None
    loan_sanction_date: date | None = None
    total_loan_amount: float | None = None
    loan_outstanding: float | None = None
    interest_paid: float | None = None
    principal_repaid: float | None = None
# HousePropertyLoanModel
